src/big_integer/multiply.rs

```diff
@@ -0,0 +1,64 @@
+/// Compute the multiplier of two strings of numbers
+///（Negative numbers and strings of numbers starting with 0 are not supported at this time）
+
```

Comment:
Maybe something like
```suggestion
/// Performs a long multiplication on numbers represented by strings.
```
Also please document, that both of the strings should actually represent non-negative numbers.